OPHD/UI/StructureInspector.cpp

```diff
@@ -132,15 +132,14 @@ void StructureInspector::update()
 
 	auto& renderer = Utility<Renderer>::get();
 
-	auto position = mRect.startPoint() + NAS2D::Vector{5, 25};
 	if (mStructure == nullptr)
 	{
-		drawLabelAndValue(position, "NULLPTR!", "");
+		text("NULLPTR!");
```

Comment:
Wondering if this should be treated as an exception? Would be a different change set but something to note.

Reply:
I agree an exception would be better. I can make that change in near future.